### ISSUE TITLE
[import/newline-after-import] `considerComments` not working with `require` statements

### ISSUE THREAD

**Issue by Arkellys** [import-js/eslint-plugin-import#2905]
The option `considerComments` doesn't seem to work with `require` statements. For example, with the following configuration:

```js
"import/newline-after-import": ["warn", { count: 2, considerComments: true }]
```

These are considered valid:

```js
const fs = require("fs");

// Test comment
const FOO = true;
```
```js
const fs = require("fs");
/**
 * Test comment
 */
const FOO = true;
```

While they should be:
```js
const fs = require("fs");


// Test comment
const FOO = true;
```
```js
const fs = require("fs");


/**
 * Test comment
 */
const FOO = true;
```

It works as it should with `import` statements.
